language/proto/config.go

```diff
@@ -174,14 +184,16 @@ func (_ *protoLang) RegisterFlags(fs *flag.FlagSet, cmd string, c *config.Config
 	// this is set for compatibility with older versions.
 	fs.Var(&modeFlag{&pc.Mode}, "proto", "default: generates a proto_library rule for one package\n\tpackage: generates a proto_library rule for for each package\n\tdisable: does not touch proto rules\n\tdisable_global: does not touch proto rules and does not use special cases for protos in dependency resolution")
 	fs.StringVar(&pc.groupOption, "proto_group", "", "option name used to group .proto files into proto_library rules")
+	fs.StringVar(&pc.stripImportPrefix, "proto_strip_import_prefix", "", "When set, .proto source files in the srcs attribute of the rule are accessible at their path with this prefix cut off.")
```

Comment:
If we require the strip prefix to be absolute, we should probably drop this flag and rely on the directive. There would be nothing to strip in the root directory where this would apply.